### ISSUE TITLE
Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by dvdnhm** [gitextensions/gitextensions#6387]
## Current behavior

GitHub > Create Pull Request menu throws 'Object reference not set to an instance of an object' error.

## Expected behavior

Proper form to create a pull request

## Steps to reproduce

Just follow the menu..

## Screenshots

![image](https://user-images.githubusercontent.com/6667615/54691045-fe483280-4af8-11e9-8b7e-8c6be7aadfb2.png)

## Did this work in the previous version of GitExtensions

I am not sure, didn't try before

## Environment

- Git Extensions 3.0.2.5232
- Build 23b6f51905006ccdde8701591df706284b4155dc
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3362.0
- DPI 120dpi (125% scaling)

## Diagnostics
File name:   C:\Program Files\Git\bin\git.exe
Arguments:   remote
Working dir: T:\xxxx\xxxx
Process ID:  2280
Started at:  2019-03-20T10:17:20.5987981-04:00
UI Thread?:  True
Duration:    71.17 ms
Exit code:   0

**Comment by dvdnhm** [comment]
It is an organization repo, does that matter.
PS: View pull requests options gives the same error and looks like the problem is related to getting branch list.
![image](https://user-images.githubusercontent.com/6667615/54694062-05be0a80-4afe-11e9-95a7-94cd14248981.png)

**Comment by pmiossec** [comment]
Is the OAuth Github token set in the settings?

And also don't forget to give us the stacktrace ....

**Comment by dvdnhm** [comment]
OAuth token is not set, I thought the process was complete but when I tried again it was stuck on this screen:
![image](https://user-images.githubusercontent.com/6667615/54695721-da88ea80-4b00-11e9-8127-2c645c48b877.png)

**Comment by pmiossec** [comment]
I'm not sure if it is linked to https://github.com/gitextensions/gitextensions/issues/3712#issuecomment-313185605

**Comment by dvdnhm** [comment]
no, it wasn't linked yet and opening the page in chrome worked, it is all good now thanks.

a check the github menu items can be added:
- if github plugin is not linked, make the menu items disabled and add 'link your Github' account menu item.

**Comment by pmiossec** [comment]
>a check the github menu items can be added:

indeed, something like that could be done. Now, who will be enough motivated to do it...
